speechbrain/lobes/models/huggingface_transformers/discrete_ssl.py

```diff
@@ -56,38 +60,37 @@ class DiscreteSSL(nn.Module):
     >>> ssl_layer_num = [7,23]
     >>> deduplicate =[False, True]
     >>> bpe_tokenizers=[None, None]
-    >>> kmeans_repo_id = "speechbrain/SSL_Quantization"
-    >>> kmeans_dataset = "LJSpeech"
+    >>> vocoder_repo_id = "speechbrain/hifigan-wavlm-k1000-LibriTTS"
```

Comment:
I tried to run the example in the docstring but I'm getting: https://huggingface.co/api/models/speechbrain/hifigan-wavlm-k1000-LibriTTS/revision/main
Also, `hubert-large-ll60k` is used as SSL so I guess the vocoder should be something like `hifigan-hubert` rather than `hifigan-wavlm`

Reply:
I updated the docstring, `hubert-large-ll60k` should be `wavlm-large`. The issue here is that the vocoder has not yet been uploaded to the SpeechBrain HF repository and is currently only available on my personal repository, I will move it.